.github/how_to_use_github_to_contribute_to_gudhi.md

```diff
@@ -0,0 +1,116 @@
+# How to use github to contribute to gudhi
+
+Similar information is available in many places:
+* https://jarv.is/notes/how-to-pull-request-fork-github/ (this one is using `upstream/master` when creating a new branch)
+* https://help.github.com/en/github/getting-started-with-github/fork-a-repo
+* https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/
+* https://gist.github.com/Chaser324/ce0505fbed06b947d962
+* etc
+
+## Get a github account
+I assume the account is called **LOGIN**, please replace as appropriate below. Log in to github.com using this account.
+
+## Fork GUDHI/gudhi-devel project
+Go to https://github.com/GUDHI/gudhi-devel and click on **fork** (top right).
+Feel free to also click on the star next to it to show you like the project!
+You can see your fork at https://github.com/LOGIN/gudhi-devel
+
+## Create a local clone on your computer
+```bash
+git clone https://github.com/LOGIN/gudhi-devel.git
+```
+
+This creates a directory gudhi-devel, which you are free to move around or rename. For the following, change to that directory:
+```bash
+cd gudhi-devel
+```
+
+## Submodule
+An interface to Hera for Wasserstein distance is available on an external git repository.
+Everytime you checkout master or merge from master, afterwards, you will need to run the command:
```

Comment:
Uh no, not that often. You only need to run this command once (while in a branch recent enough to have hera). Then you don't need to run it again until the version of hera we depend on changes, or we add more external dependencies. I think I would put this as a second line in "create a local clone", with a reminder near "git fetch".